src/CliRequest.php

```diff
@@ -96,6 +96,7 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
             if (strtoupper($method) === 'GET') {
                 $this->request = \Slim\Http\Request::createFromEnvironment(\Slim\Http\Environment::mock([
                     'REQUEST_METHOD'    => 'GET',
+                    'HTTPS'             => 'on'
```

Comment:
Although I don't see how this would break BC, I would still prefer for it to be injected as a dependency through the constructor. 
That way you can add a test just for this and all other tests will pass too.